libs/template/renderer_test.go

```diff
@@ -69,6 +57,10 @@ func assertBuiltinTemplateValid(t *testing.T, template string, settings map[stri
 	err = renderer.persistToDisk(ctx, out)
 	require.NoError(t, err)
 
+	// Verify permissions on file and directory
+	testutil.AssertFilePermissions(t, filepath.Join(tempDir, "my_project/README.md"), fs.FileMode(0o644))
```

Comment:
It looks like permissions for Windows become 666 and 777 instead now, any ideas why?